core/templates/pages/exploration-editor-page/suggestion-modal-for-editor-view/exploration-editor-suggestion-modal.template.html

```diff
@@ -26,7 +26,7 @@ <h3>Review Suggestion</h3>
   </div>
   <div class="oppia-review-message-text" ng-show="canAccept">
     Brief Description of Changes (required if accepting):
-    <input class="protractor-test-suggestion-commit-message w-100" type="text" ng-model="commitMessage" maxlength="1000">
+    <input class="protractor-test-suggestion-commit-message w-100" type="text" ng-model="commitMessage" maxlength="375">
```

Comment:
Considering we have the same limit in multiple files. Let's use a constant in all the places.

Reply:
done